stackle_api/app/routes.js

```diff
@@ -14,10 +14,9 @@ const Reply = postModels.Reply;
 
 module.exports = function (app, db) {
 
-	//api
-	//TODO later
```

Comment:
??

Reply:
@tharindupr  On requesting localhost:8080/login , user will be redirected to credentials page where enduser will enter his GitHub credentials and authorize the app. On success he will be issued a token. Protected resources can only be access using this token.